benchmark/venv.py

```diff
@@ -162,7 +162,12 @@ def create_virtualenv():
     root = ROOT_DIR
     venv_name = virtualenv_name()
     venv_path = os.path.join(root, 'venv', venv_name)
-    venv_python = os.path.join(venv_path, 'bin', 'python')
+    python_executable = os.path.basename(sys.executable)
+    # venv puts the python executable under Scripts instead of bin on windows
+    if os.name == "nt":
+        venv_python = os.path.join(venv_path, 'Scripts', python_executable)
+    else:
+        venv_python = os.path.join(venv_path, 'bin', python_executable)
```

Comment:
On OS X, the interpreter executable in the build directory may either be 'python' or 'python.exe' depending on the file system in use.  But, when installed or in a virtual environment like the case here, it is always some form of 'python', not 'python.exe'.